homeassistant/components/cover/__init__.py

```diff
@@ -167,6 +167,15 @@ async def async_unload_entry(hass, entry):
 class CoverEntity(Entity):
     """Representation of a cover."""
 
+    def __init__(self):
+        """Initialize the cover."""
+        self._is_last_toggle_direction_open = True
```

Comment:
I think we can do like this, then there's no need to update all cover implementations to call `super().__init__()`:

```suggestion
    self._is_last_toggle_direction_open = None
```

Reply:
Wouldn't this be a class variable which will change from any object.

If i toggle the cover in the kitchen to go up and then the cover in the livingroom which would go down and then the cover in the kitchen again and it would go up again (but i want it to go down)

Reply:
When the class variable is assigned to from an instance, `self._is_last_toggle_direction_open = True`, that instance now has its own instance variable which has priority over the class variable.

Reply:
When does your IDE complain, both when the class variable is there and when it isn't?

I don't think this is non-nice coding practice for Python, we use this pattern already, for example in the base Entity:
https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity.py#L87-L100

Reply:
IDE was complaining, solved it on my side. So it will be a class variable!